### PR TITLE
feat(crons): Add environment support to the HTTP examples

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Will do SDKs in follow up PRs

### CODE REVIEW

File: src/platforms/common/crons/index.mdx [hunk 1]
```diff
@@ -109,6 +109,42 @@ If you expect your scheduled jobs to overlap, read about [Overlapping Jobs](#ove
 
 </Note>
 
+### Specifying monitor environments (Optional)
+
+When sending check-ins to your monitor you may specify the `environment` of the
+check-in. This allows you to monitor a single schedule across multiple
+environments.
+
+If you do not specify an environment with your check-ins the default is `production`.
```
Comment (shanamatthews) [inline src/platforms/common/crons/index.mdx:118]:
```suggestion
If you don't specify an environment with your check-ins the default is `production`.
```

File: src/platforms/common/crons/index.mdx [hunk 2]
```diff
@@ -109,6 +109,42 @@ If you expect your scheduled jobs to overlap, read about [Overlapping Jobs](#ove
...
+
+<Alert level="info">
+
+Monitor environments are still early in development. Currently after a check-in
```
Comment (shanamatthews) [inline src/platforms/common/crons/index.mdx:122]:
```suggestion
Monitor environments are still early in development. Currently, after a check-in
```

**Review by shanamatthews — APPROVED** [review]
looks great except for your bonkers mid-sentence linebreaks 🤢 
jk (not really)

**Comment by evanpurkhiser** [comment]
The line breaks are to make it easier to read in my editor :)

**Pull request merged**